bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/GemsStrategy.scala

```diff
@@ -61,38 +59,36 @@ trait GemsStrategy extends AgsStrategy {
       OdgwFlexureId    -> GsaoiOdgw.Group.instance
     )
 
-    val adjustedConstraints = catalogQueries(ctx, mt).map { constraint =>
-      // Adjust the magnitude limits for the conditions.
-      val adjustedMagConstraints = constraint.magnitudeConstraints.map(c => c.map(m => ctx.getConditions.magAdjustOp().apply(m.toOldModel).toNewModel))
-      constraint.withMagnitudeConstraints(adjustedMagConstraints)
-    }
+    // Maps from IDs to guide probe band
+    def GuideProbeBandMap = Map[Int, MagnitudeBand](
+      CanopusTipTiltId -> MagnitudeBand.R,
+      OdgwFlexureId    -> MagnitudeBand.H
+    )
```

Comment:
Should this be `R` or the new bands list thing?  Should `r` and `UC` not work as well?